src/test/model/processing/test_SingleLogitBiogemeConfig.py

```diff
@@ -0,0 +1,157 @@
+                FunctionalExpression('ASC_TRAIN + B_TIME * TRAIN_TT_SCALED + B_COST * TRAIN_COST_SCALED'),
+                availability_condition=FunctionalExpression('TRAIN_AV_SP'),
+                choice_idx=1),
+            'alt2': Alternative(
+                FunctionalExpression('ASC_SM + B_TIME * SM_TT_SCALED + B_COST * SM_COST_SCALED'),
+                availability_condition=FunctionalExpression('SM_AV'),
+                choice_idx=2),
+            'alt3': Alternative(
+                FunctionalExpression('ASC_CAR + B_TIME * CAR_TT_SCALED + B_COST * CAR_CO_SCALED'),
+                availability_condition=FunctionalExpression('CAR_AV_SP'),
+                choice_idx=3)
+        }
+
+        choice = FunctionalExpression('CHOICE')
+
+        raw_data = pd.read_csv(f'{os.path.dirname(__file__)}/../../resources/swissmetro.csv', sep='\t')
+        raw_data = raw_data.drop(
+            raw_data[((raw_data['PURPOSE'] != 1) * (raw_data['PURPOSE'] != 3) + (raw_data['CHOICE'] == 0)) > 0].index)
+        data = Data(raw_data, None, derivatives)
+        model = Model(data, alternatives, choice)
+        config = SingleLogitBiogemeConfig()
+
+        evaluation = config.process(model)
+
+        self.assertEqual(type(evaluation.result), pd.DataFrame)
+
+    @staticmethod
+    def __example__b01logit() -> Evaluation:
+        """
+        Example for using biogeme.
+        Source: https://github.com/michelbierlaire/biogeme/blob/master/examples/swissmetro/b01logit.py (06.07.2023)
+        """
+        #%%
+        import biogeme.biogeme as bio
+        from biogeme import models
+        from biogeme.expressions import Beta
+        import pandas as pd
+        import biogeme.database as db
+        from biogeme.expressions import Variable
+        #%%
+        # Read the data
+        df = pd.read_csv('src/test/resources/swissmetro.dat', sep='\t')
+        database = db.Database('swissmetro', df)
+
+        PURPOSE = Variable('PURPOSE')
+        CHOICE = Variable('CHOICE')
+        GA = Variable('GA')
+        LUGGAGE = Variable('LUGGAGE')
+        TRAIN_CO = Variable('TRAIN_CO')
+        CAR_AV = Variable('CAR_AV')
+        SP = Variable('SP')
+        TRAIN_AV = Variable('TRAIN_AV')
+        TRAIN_TT = Variable('TRAIN_TT')
+        SM_TT = Variable('SM_TT')
+        CAR_TT = Variable('CAR_TT')
+        CAR_CO = Variable('CAR_CO')
+        SM_CO = Variable('SM_CO')
+        SM_AV = Variable('SM_AV')
+        MALE = Variable('MALE')
+        GROUP = Variable('GROUP')
+        TRAIN_HE = Variable('TRAIN_HE')
+        SM_HE = Variable('SM_HE')
+        INCOME = Variable('INCOME')
+        # Removing some observations can be done directly using pandas.
+        # remove = (((database.data.PURPOSE != 1) &
+        #           (database.data.PURPOSE != 3)) |
+        #          (database.data.CHOICE == 0))
+        # database.data.drop(database.data[remove].index,inplace=True)
+        # Here we use the "biogeme" way:
```

Comment:
maybe delete the commented out code but I am not sure how important that is in the test folder